demos/demo_operation_log.sh

```diff
@@ -27,10 +27,10 @@ comment "The repo now looks like this:"
 run_command "jj log"
 comment "The most recent portion of the operation log
 is:"
-run_command "jj op log --color=always | head"
+run_command_allow_broken_pipe "jj op log | head"
```

Comment:
TODO: Change this to `jj op log --limit 4`.

Not sure whether I should also delete `run_command_allow_broken_pipe`, since it documents a gotcha and could easily become useful later.

Reply:
I don't remember whether `jj op log --limit` works in 0.9.0, but we could merge this together with the next release.

Reply:
Do you think we should even change the exit code from 3 to 0 for broken pipes?

Reply:
I vaguely recall a discussion about this somewhere when the broken pipe error code was implemented.

I see no reason for a non-zero exit code on a broken pipe, but I'm not sure how UNIXy that is.